### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by Estherstarr**
Mobile Portfolio Version Skeleton
- Creat a sample for my Mobile Portfolio using index.html and CSS
- Add  .gitignore to ignore node_modules and test.md files
- Include Linters for HTML/CSS
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  </head>
+  <body>
+    <div class="container">
+        <header class="tool-bar">
+          <div class="tools">
+            <a href="/" class="logo">Onomeh </a>
+            <nav class="navbar">
+              <button class="nav-button" type="button">
+                <img src="assets/Icons/hamburger-button.svg" alt="more" />
+              </button>
```
Comment (thecodechaser) [inline index.html]: - [ ] You did a very great job on the design but there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your hamburger icon should be on the right of the screen but currently, it is appearing in the center. Please place your hamburger icon right side of the screen and make sure it matches the design. 

Your design

Screenshot (153)

Figma design

Screenshot (152)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Estherstarr 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
-  Clean code ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
Screenshot from 2022-02-17 17-52-19

@thecodechaser This image is how it displays on my browser. The hamburger is on the right.  I'm a bit confused about what to do next

**Comment by thecodechaser** [comment]
@Estherstarr The screenshot you shared is the desktop version, You are currently seeing your webpage on desktop screen
Use inspect elements to switch to mobile screen and you will see the icon is in the center

**Comment by Estherstarr** [comment]
@thecodechaser what do I need to do. Because the instruction asked to use flexbox. Should I use media queries? I'm stuck can you give me hints on how to go about this error

**Comment by thecodechaser** [comment]
@Estherstarr Hello there, You go with one of the following solutions
- Apply `margin-left` to your navbar accordingly so it matches the design
- You can use `position: absolute;` for your hamburger button and apply top and right properties accordingly so it matches the design.

**Comment by Estherstarr** [comment]
Thank you @thecodechaser 

The requested changes made
-The hamburger  icon is on the right side of the screen with or without inspect

File: README.md [hunk 2]
```diff
@@ -1 +1,98 @@
...
+- LinkedIn: [LinkedIn](https://linkedin.com/in/onwuanibeonome)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
```
Comment (elmar8287) [inline README.md]: - [ ] [OPTIONAL] As it was not marked by the previous review, this is marked as an optional suggestion: 
- Please, could you remove the doubling of information in README file 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      
+   <a href="/" class="logo">My Logo</a>
+              <nav class="navbar">
+                <button class="nav-button" type="button">
+                  <img src="assets/Icons/hamburger-button.svg" alt="more" />
+                </button>
+              </nav>                      
+       
+      </div>
+
```
Comment (elmar8287) [inline index.html]: - [ ] [OPTIONAL] As it was not marked by the previous review, this is marked as an optional suggestion: 
- Please, could you put your toolbar section into the `header` tag - it is for considering HTML semantic structure 👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            ><img src="./assets/Icons/github.svg" alt="Github icon"
+          /></a>
+        </li>
+        <li class="angel-icon">
+          <a title="angel" href="#"
+            ><img src="./assets/Icons/angel-list.svg" alt="Angel icon"
+          /></a>
+        </li>
+      </ul>
+    </section>
```
Comment (elmar8287) [inline index.html]: - [ ] [OPTIONAL] As it was not marked by the previous review, this is marked as an optional suggestion: 
- Please, consider closing the `body` and `HTML` tags 👍

**Review by elmar8287 — APPROVED** [review]
Hi, @Estherstarr.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

## Please, before merging consider my OPTIONAL suggestion in the comments under the review 👍 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Estherstarr** [comment]
@elmar8287 Thank you for the review 
Optional changes 
 - closing the body and HTML tags
 - Add toolbar section into the header tag 
 -  Remove the doubling of information in README file

**Pull request merged**